src/pages/Detail/CalenderComp.js

```diff
@@ -1,12 +1,17 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import { withRouter } from 'react-router-dom';
 import styled from 'styled-components';
 import DatePicker from 'react-datepicker';
 import 'react-datepicker/dist/react-datepicker.css';
 
 const CalenderComp = props => {
   const [startDate, setStartDate] = useState(new Date());
-  const [dateValue, setDateValue] = useState('');
+  const [theDate, setTheDate] = useState('');
+
+  useEffect(() => {
+    const dateValue = theDate;
+    props.transferValue(dateValue);
```

Comment:
- dataValue 라는 변수를 추가적으로 만드실 필요가 없어보입니다!
```suggestion
    props.transferValue(theDate);
```
- 그리고 props.transferValue 라고 사용하시기 보단, props 자리에서 바로 구조분해할당 하신 뒤에 transferValue 만 사용해주시면 좋을 것 같습니다!